### PR TITLE
[FLINK-31748] Dummy implementation to fix compilation failure

### PR DESCRIPTION
**Pull request by tisonkun**
TRIVIAL

### CODE REVIEW

File: .github/workflows/daily.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 name: Nightly
 on:
+  pull_request:
```
Comment (tisonkun) [inline .github/workflows/daily.yml:21]: To be reverted after verified.

Comment (tisonkun) [reply .github/workflows/daily.yml:21]:
```suggestion
```

Verified. Reverted.

File: flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/reader/PulsarSourceFetcherManager.java [hunk 2]
```diff
@@ -86,6 +86,11 @@ public void addSplits(List<PulsarPartitionSplit> splitsToAdd) {
         }
     }
 
+    // @Override // to keep compatible with Flink 1.17
+    public void removeSplits(List<PulsarPartitionSplit> splitsToRemove) {
+        // empty - wait for FLINK-31748 to implement it.
```
Comment (reswqa) [inline flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/reader/PulsarSourceFetcherManager.java:91]: Maybe we can add a `TODO` tag here.

**Review by reswqa — APPROVED** [review]
Thanks @tisonkun, LGTM.

File: flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/reader/PulsarSourceFetcherManager.java (hunk 2 above)
Comment (tisonkun) [reply flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/reader/PulsarSourceFetcherManager.java:91]:
```suggestion
        // TODO empty - wait for FLINK-31748 to implement it.
```

**Review by reswqa — APPROVED** [review]

**Comment by tisonkun** [comment]
Merging...

**Pull request merged**